### ISSUE TITLE
Add optional setting to degrade factory health with use

### ISSUE THREAD

**Issue by WildWeazel** [Civcraft/FactoryMod#13]
Probably a global toggle plus a per-recipe health cost applied when activated. If not enough health, cancel and warn user.

**Comment by GavJenks** [comment]
I'm not sure I see the point of this. It equates to literally the same thing as just adjusting all of the recipes to be slightly worse at the end of the day. If you want to do that, you should just do that, and it would be simpler to understand and more intuitive. (Though I'm not sure why you would want to do that, the plugin seems pretty well balanced)

**Comment by WildWeazel** [comment]
Not necessarily, because it makes factory maintenance something to be managed by the owner. Rolling it into the recipes forces whoever runs each job to pay up front. Separating it from recipes allows users to take advantage of public or group owned factories at someone else's expense, providing for slightly more interesting management. It's no longer advantageous for as many people as possible to share one factory and run as many jobs as possible as quickly as possible. Owners might be more likely to restrict access or introduce membership fees, production taxes, etc.

**Comment by GavJenks** [comment]
I find it very difficult to believe that anybody owning an expensive factory would just shrug and eat that cost, versus the obvious response of "Hey guys, if you use the factory, there is a mandatory fee of ___ to cover the wear and tear. Put it in the fees chest next to the factory. Or you will lose your privileges"

Which you mentioned in your post just now, but again, all of those things (taxes, fees, etc.) simply end up being identical to an increased recipe cost, except much more tedious, annoying, and complicated. So just make the same change more elegantly and less pointlessly micromanagey by increasing the recipe cost, if that's what you want to do.

**Comment by ProgrammerDan** [comment]
Assign this to me too ... sensing a pattern. I agree with @WildWeazel that this is an interesting mechanic. I say that as a factory manager who runs a public institution full of 90% of the factories possible within the tech tree. This would be a sea change for us if all factories converted to this scheme, but it would be *far more interesting* in terms of the overall social mechanics. Just keeping things simple isn't a valid goal. Fostering interesting interactions and forcing innovation in design and execution *is* a valid goal, and this feature would allow such interactions to occur, whereas right not it's not possible.

There is no motivation whatsoever to restrict factory access beyond simply making money, and diamond mining is far more profitable. If letting people use my factories was actively costing me *extra* money, though, I would charge for their use and potentially hire help, based on those earnings, to keep factories healthy. The options are fascinating and complex, and in those complexities lives some real opportunities for fun new social interactions. Given that the installation of "my" factories was through 50% or more public donation, I'm still motivated to keep access relatively open, it just changes how that access is moderated.
